tests/Unit/Domain/DomainTestHelpers.cpp

```diff
@@ -3,18 +3,214 @@
 
 #include "tests/Unit/Domain/DomainTestHelpers.hpp"
 
+#include <algorithm>
 #include <catch.hpp>
 #include <typeinfo>
 
 #include "Domain/Block.hpp"
 #include "Domain/BlockNeighbor.hpp"
+#include "Domain/CoordinateMaps/AffineMap.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
 #include "Domain/Direction.hpp"
 #include "Domain/Domain.hpp"
 #include "Domain/ElementId.hpp"
 #include "Domain/SegmentId.hpp"
+#include "Utilities/ConstantExpressions.hpp"
 #include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/MakeVector.hpp"
```

Comment:
I don't see any uses of the new includes AffineMap, ProductMaps, or MakeVector.  Remove them unless I'm missing something.